scripts/mkfiles.sh

```diff
@@ -438,11 +444,14 @@ echo "====================================================================="
 
 mkdir -p run
 
+# Specify the RTS flags for capturing GHC runtime stats into a file when the program exits
+RTSFLAGS="-N8 --disable-delayed-os-memory-return -I0.3 -Iw600 -A16m -F1.5 -H2500M -T -t --machine-readable -Sghc-rts-"
```

Comment:
Nice to make a variable out of the RTSFLAGS.  I haven't spent the time to research the meaning of all the params, so what you're suggesting is probably a good place to start.  My only concern is the CPU cores setting of N8.  If somebody is running a quad core raspberry pi, would this setting be incompatible?

Reply:
Yeah, this is VERY specific to my machine, and the ```N``` flag should definitely not be hardcoded like I have it here, lol.  I did try to run it with just ```-N```, and it worked. When I inspected the RTS flags I saw it set to 16, so I am not sure if it was set _well_.  One thing I was thinking is provide a default set of RTS flags but also expose an environment variable that someone could set all the RTS flags for, then they can tune them as they see fit.  Its a bit kludgy, but it also doesn't blow up the complexity in the mkfiles shell script so it should be pretty easy to implement.